api_blueprint/group.apib

```diff
@@ -164,3 +164,24 @@ Create new group
             transfer-encoding: chunked
 
     + Body
+
+# Group Users [/api/v1/groups/{group_id}/users]
+
++ Parameters
+    + group_id: 1 (required, number) - ID of the Group in form of an integer
+
+## Add user [POST]
+
++ user_id (required, number) - ID of the User in form of an integer
++ access_token (required, string) - Token of acting user (Require user with admin or group admin level)
```

Comment:
@walbertus should the request put access token in header instead? I think our get_token method in base controller do support token in header

Reply:
There are 3 ways to put an access token:
access_token param
token param
Authorization header

These ways evaluated from top to bottom. When people use Authorization it can be overridden by access_token param, this is why I'm not sure which one is default so I put the access_token param.

Should I set the Authorization header as default?

Reply:
yes I think the better practice is to put it there